src/MoBi.Presentation/Presenter/EditModuleConfigurationsPresenter.cs

```diff
@@ -98,6 +103,7 @@ public void RemoveModuleConfiguration(ITreeNode selectedTreeNode)
          _view.RemoveNodeFromSelectedView(selectedTreeNode);
 
          addModuleToSelectionView(moduleConfigurationDTOFor(selectedTreeNode).Module);
+         ViewChanged();
```

Comment:
explain why this is required. We never put comment there but I think this is helpful